sharding-core/sharding-core-preprocessor/src/test/java/org/apache/shardingsphere/core/preprocessor/segment/select/pagination/engine/TopPaginationContextEngineTest.java

```diff
@@ -71,6 +75,47 @@ public void assertCreatePaginationContextWhenRowNumberPredicatePresentAndOperato
         assertCreatePaginationContextWhenRowNumberPredicatePresentAndWithGivenOperator(">=");
     }
     
+    @Test
+    public void assertCreatePaginationContextWhenPredicateInRightValue() {
+        String name = "rowNumberAlias";
+        ColumnSegment columnSegment = new ColumnSegment(0, 10, name);
+        PredicateSegment predicateSegment = new PredicateSegment(0, 10, columnSegment, mock(PredicateInRightValue.class));
+        AndPredicate andPredicate = new AndPredicate();
+        andPredicate.getPredicates().add(predicateSegment);
+        Collection<AndPredicate> andPredicates = Collections.singleton(andPredicate);
+        ProjectionsContext projectionsContext = mock(ProjectionsContext.class);
+        when(projectionsContext.findAlias(anyString())).thenReturn(Optional.of("predicateRowNumberAlias"));
+        PaginationContext paginationContext = topPaginationContextEngine.createPaginationContext(new TopSegment(0, 10, "text", null, name), andPredicates, Collections.emptyList());
+        Optional<PaginationValueSegment> paginationValueSegmentOptional = paginationContext.getOffsetSegment();
```

Comment:
We prefer do not name `xxxOptional`, just name as `xxx`.
So `paginationValueSegmentOptional` should be `paginationValueSegment`